patches/net/minecraft/world/flag/FeatureFlagRegistry.java.patch

```diff
@@ -0,0 +1,44 @@
+--- a/net/minecraft/world/flag/FeatureFlagRegistry.java
++++ b/net/minecraft/world/flag/FeatureFlagRegistry.java
+@@ -75,6 +_,18 @@
+         }, p_249796_ -> List.copyOf(this.toNames(p_249796_)));
+     }
+ 
++    public FeatureFlag getFlag(ResourceLocation name) {
```

Comment:
Would it be worth while patching in a static helper for looking up flags by name? `FeatureFlags.byName(<id>)`. Using the registry isn't immediately clear that this is how modders need to be looking up their custom flags.

Reply:
I'm not sure that's needed. This is an advanced use case, if a mod is gonna use this, they should have made themselves properly familiar with the system and understood its implications. Apart from that, this needs docs anyway once it's done.

Reply:
Thats understandable, I was thinking of less advanced users using the system, its not immediately clear to them how to obtain their registered flags. But yeah since it is a more intermediate-advanced feature set its understandable to assume users would already know this.